transport-native-kqueue/src/main/java/io/netty/channel/kqueue/Native.java

```diff
@@ -111,7 +111,8 @@ private static void loadNativeLibrary() {
         if (!name.startsWith("mac") && !name.contains("bsd") && !name.startsWith("darwin")) {
```

Comment:
Should this be replaced with calls to PlatformDependent to get this information? And missing ones added there?

Reply:
maybe... I did want to keep the change focused

Reply:
Ok no worries any obj if i go do this after then? new pr post merge of this?

Reply:
let us merge this one first and then we can see if it makes sense to move some of the stuff (I think it may make sense tho ;))